newrelic.yml

```diff
@@ -459,6 +459,12 @@ common: &default_settings
   # If true, the agent obfuscates Mongo queries in transaction traces.
   # mongo.obfuscate_queries: true
 
```

Comment:
The instrumentation generator should probably be updated to add the config entry to this file too. I can take that todo if you'd like!

Reply:
```suggestion

# Controls auto-instrumentation of the elasticsearch library at start up. 
# May be one of [auto|prepend|chain|disabled].
# instrumentation.elasticsearch: auto

```

Reply:
(Might be too many spaces?? I can't tell)